pages/api/commodityPrice.ts

```diff
@@ -0,0 +1,55 @@
+import nc from 'next-connect';
+import cors from 'cors';
+import axios from 'axios';
+
+type SpreadProfile = {
+	spreadProfile: string;
+	bidSpread: number;
+	askSpread: number;
+	bid: number;
+	ask: number;
+};
+
+type PriceResponse = {
+	spreadProfilePrices: SpreadProfile[];
+	topo: {
+		platform: string;
+		server: string;
+	};
+	ts: number;
+};
+
+const COMMODITIES_BASE_API_URL =
+	'https://forex-data-feed.swissquote.com/public-quotes/bboquotes/instrument';
```

Comment:
should this be imported from [here](https://github.com/Kwenta/kwenta/pull/804/files?diff=unified&w=1#diff-565a6111eb3ea1bc8df86664232eddde4daa587826c805246f81b8ccbe534518R11-R12)